test/basic.test.js

```diff
@@ -5,9 +5,16 @@ const os = require('os')
 const test = require('tap').test
 const pino = require('pino')
 const dateformat = require('dateformat')
+const path = require('path')
+const rimraf = require('rimraf')
```

Comment:
Since we only support Node 12+, rimraf is redundant, you can use `fs.rmSync(dir, { recursive: true, force: true });`

Reply:
I'm not doing that because it changed again in v16, so this move it a bit more far out in the future :(.